coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/transformers/GenericCoralRegistryOperatorRenameSqlCallTransformer.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.transformers;
+
+import org.apache.calcite.sql.SqlCall;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import com.linkedin.coral.com.google.common.base.CaseFormat;
+import com.linkedin.coral.com.google.common.base.Converter;
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+import com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry;
+
+
+public class GenericCoralRegistryOperatorRenameSqlCallTransformer extends SqlCallTransformer {
```

Comment:
nit: please add a small description and an example.

Reply:
Done.